docs/source/changelog/0.9.0-changelog.rst

```diff
@@ -0,0 +1,256 @@
+******
+v0.9.0
+******
+
+:Date: August 01, 2021
+
+Contributors
+============
+
+A total of 35 people contributed to this
+release. People with a '+' by their names authored a patch for the first
+time.
+
+* Alex Lembcke
+* Benjamin Hackl
+* Darylgolden
+* Devin Neal
+* Harivinay +
+* Hugues Devimeux
+* Jared Hughes +
+* Jason Villanueva
+* Kadatatlu Kishore +
+* LED Me Explain +
+* Laith Bahodi
+* Mohammad Al-Fetyani
+* Noam Zaks
+* Oliver
+* PaulCMurdoch
+* Raghav Prabhakar +
+* Ryan McCauley
+* Suhail Sherif +
+* Taektiek +
+* Udeshya Dhungana +
+* UraniumCronorum +
+* Vinh H. Pham (Vincent) +
+* ccn +
+* icedcoffeeee +
+* sahilmakhijani +
+* sparshg
+
+
+The patches included in this release have been reviewed by
+the following contributors.
+
+* Abhijith Muthyala
+* Alex Lembcke
+* Benjamin Hackl
+* Darylgolden
+* Devin Neal
+* Harivinay
+* Hugues Devimeux
+* Jan-Hendrik Müller
+* Jason Villanueva
+* KingWampy
+* Laith Bahodi
+* Lino
+* Mohammad Al-Fetyani
+* Oliver
+* Raghav Goel
+* Suhail Sherif
+* icedcoffeeee
+* sahilmakhijani
+* sparshg
+
+Pull requests merged
+====================
+
+A total of 50 pull requests were merged for this release.
+
+Deprecated classes and functions
+--------------------------------
+
+* `#1848 <https://github.com/ManimCommunity/manim/pull/1848>`__: Deprecated parameters for :class:`~.DashedLine` and :class:`~.DashedVMobject`
+   - ``dash_spacing`` is an unused parameter
+   - ``positive_space_ratio`` has been replaced with the shorter name ``dashed_ratio``
+
+* `#1773 <https://github.com/ManimCommunity/manim/pull/1773>`__: Remove all classes and functions that were deprecated until `v0.7.0` and `v0.8.0`
+
+
+New features
+------------
+
+* `#1677 <https://github.com/ManimCommunity/manim/pull/1677>`__: New Mobject: Added :class:`~.Tabular`
```

Comment:
```suggestion
* `#1677 <https://github.com/ManimCommunity/manim/pull/1677>`__: New Mobject: Added :class:`~.Table`
```

Reply:
That will be reflected when he regenerates the changelog. I changed the title of the PR so it will be updated

Reply:
Oh, cheers!